ynput/ayon-core
# Publisher: Add plugin details widget

## Changelog Description
Show basic plugin details in report.

## Additional information
Took over the PR and changed it a little. Each tab in publish report viewer does refresh content only if is set as active. The plugins details widget does show details for each plugin or for selected plugins.

At this moment shows only basic information like documentation, families, order, class name, filepath and process time. There is no visual indication if was processed, skipped, succeded or failed and there are no logs visible in UI Those features can be added in future PRs (logs might make it uber slow).

The UI can get quite slow with bigger list of plugins. I was thinking that most of the information would be not created until user clicks on expand button or change to advanced mode.

### Screenshots
![image](https://github.com/user-attachments/assets/5454aa94-87de-4084-90af-0713eb6451ae)


## Testing notes:
1. Open publisher in host of your choice (traypublisher is easiest).
2. Go to `Details` tab.
3. Change tab on right side to `Plugins Details`
4. You should see cards with details about plugins.
